library/src/main/java/com/getstream/sdk/chat/viewmodel/ChannelViewModel.java

```diff
@@ -146,8 +158,8 @@ public ChannelViewModel(Application application, Channel channel) {
         return channelState;
     }
 
-    public LiveData<List<Message>> getMessages() {
-        return messages;
+    public MutableLiveData<List<Message>> getMessages() {
```

Comment:
never return `MutableLiveData` from ViewModels' public methods

Reply:
I have use postValue() on the function. (please see addMessage() on 505line)
So we should return MutableLiveData

Reply:
I am sure you have a good reason but please don't do it like this; if you need to add a message then expose a public method for that. Exposing `MutableLiveData` is bad

Reply:
I see. I will change this. ;)